To Do List/js/app.js

```diff
@@ -0,0 +1,143 @@
+// Select the Elements
+const clear = document.querySelector(".clear");
+const dateElement = document.getElementById("date");
+const list = document.getElementById("list");
+const input = document.getElementById("input");
+
+// Classes names
+const CHECK = "fa-check-circle";
+const UNCHECK = "fa-circle-thin";
+const LINE_THROUGH = "lineThrough";
+
+// Variables
+let LIST, id;

```

Comment:
Good use of `const` above, `LIST` could be one too, it only gets assigned once on live 20. You could go with something like: 
`const LIST = data && JSON.parse(data) || []` 
Which would result in the same outcome as you have here. This kind of inline boolean logic is very common in js. `id` also doesn't need to be assigned in the `if` as `[].length` will return `0` so `let id = LIST.length` on the next line will work as well.
This means you can get rid of the `if(data)` block entirely and just always call `loadList(LIST)` even if the array is empty ([].forEach() doesn't break, it will do nothing as you intend here). That is a pretty standard pattern.